ksql-execution/src/main/java/io/confluent/ksql/execution/materialization/AggregatesInfo.java

```diff
@@ -60,6 +60,10 @@ public int startingColumnIndex() {
     return aggregateFunctions;
   }
 
+  int valueColumnCount() {
+    return startingColumnIndex + aggregateFunctions.size();
```

Comment:
why can't this just by part of the schema (`schema.value().size()`)? Otherwise we're baking into this code the assumption that the startingColumnIndex happens to be the number of columns that aren't aggregates (and that there aren't any columns after aggregateFunctions).

Reply:
lol. If you look where this is used, it's compared to `schema.value().size()` as a sanity check.

However, the whole `AggregateInfo` type is removed in #3910, so this is a mute point.